receiver/mongodbatlasreceiver/internal/mongodb_atlas_client.go

```diff
@@ -687,6 +687,30 @@ func (s *MongoDBAtlasClient) GetEvents(ctx context.Context, groupID string, opts
 	return events.Results, hasNext(response.Links), nil
 }
 
+// GetOrgEvents returns the events specified for the set organizations
+func (s *MongoDBAtlasClient) GetOrgEvents(ctx context.Context, orgID string, opts *GetEventsOptions) (ret []*mongodbatlas.Event, nextPage bool, err error) {
```

Comment:
Projects are associated with an organization, right? If I were a user, I might expect that getting "organization events" means getting "all events from all projects in an organization". I think we should have some language in the README at least to clarify that these are "events not associated with a project" or something to that effect.

Reply:
An organization can have many projects, so a project is under an organization. From my understanding and testing the endpoints, these organization events won't capture project specific events. Organization events deal with API keys, billing charges, and groups changes. Project events are related to host, cluster and alerts from inspection. I updated the readme.